package.json

```diff
@@ -1,8 +1,7 @@
 {
-  "name": "rrule",
-  "version": "2.6.4",
+  "name": "@pulsarplatform/rrule",
```

Comment:
Not sure if this was a stop gap given time to merge the PR but it likely cannot be merged until these changes are removed

Reply:
Also this, i removed. Thanks!